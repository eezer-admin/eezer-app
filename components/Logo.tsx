import * as React from 'react';
import { Image } from 'react-native';

export default function Logo() {
  return (
    <Image
      style={{
        width: 240,
        height: 60,
        marginBottom: 50,
      }}
      source={require('../assets/logo.png')}
    />
  );
}
