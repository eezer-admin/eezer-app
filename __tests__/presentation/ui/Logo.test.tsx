import { describe, expect, it } from '@jest/globals';
import Logo from '@src/presentation/ui/Logo';
import renderer from 'react-test-renderer';

describe('Logo', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Logo />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
