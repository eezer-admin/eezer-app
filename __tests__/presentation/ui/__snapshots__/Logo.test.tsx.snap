// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<Image
  source={1}
  style={
    {
      "height": 60,
      "marginBottom": 50,
      "width": 240,
    }
  }
/>
`;
